integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java

```diff
@@ -3422,6 +3484,36 @@ public static V1Ingress createIngress(String namespace, V1Ingress ingressBody)
     return ingress;
   }
 
+  /**
+   * Get a list of custom resource definitions deployed in the Kubernetes cluster.
+   *
+   * @return String list of crds as String of V1CustomResourceDefinitionList
+   * @throws ApiException when list fails.
+   */
+  public static String listCrds() throws ApiException {
```

Comment:
This is only listing the VZ crds, right? If yes, can you change the method name to be more VZ specific?

Reply:
Even though vz specific CustomObjectsApi instance is used it lists all crds deployed in the cluster. You are right that the instance name is little confusing.

Actually, we can reuse the same CustomObjectsApi instance customObjectsApi for creating VZ artifacts too.
Removed the vzCustomObjectsApi instance and reusing the customObjectsApi .

So I guess since listCrds is listing all crds we don't need to change its name.

Running the tests to make sure I am not breaking anything.